lib/kube_link_generator.rb

```diff
@@ -66,17 +69,26 @@ def get_pods_for_role(role_name, job, options = {})
     end
   end
 
-  def get_exported_properties(pod, job)
-    if pod.metadata.annotations["skiff-exported-properties-#{job}"]
-      JSON.parse(pod.metadata.annotations["skiff-exported-properties-#{job}"])
-    elsif pod.metadata.annotations["skiff-exported-properties"]
-      JSON.parse(pod.metadata.annotations["skiff-exported-properties"])[job]
+  def get_exported_properties(role_name, pod, job_name)
+    if pod.metadata.annotations["skiff-exported-properties-#{job_name}"]
+      if pod.metadata.annotations["skiff-exported-digest-#{job_name}"]
+        # Copy the digest over, so that if the source role changes we can be restarted
```

Comment:
[nit] Punctuation.

Reply:
Fixed in da315a15408eec9730f51d8d295c513d19f22eb2